layouts/states/register-by-mail.html

```diff
@@ -0,0 +1,69 @@
+<div class="usa-grid page-inner">
+  <header>
+    <h1 class="usa-display">Register to vote</h1>
+  </header>
+  <!-- Steps BEGIN -->
+  <div class="graphic-list">
+    <img src="{{ .Site.BaseURL }}/assets/img/step-1.png">
+    <h4 class="usa-graphic-list-heading">Step 1</h4>
+    <p>Fill out your voter registration.</p>
+    <img src="{{ .Site.BaseURL }}/assets/img/step-2.png">
+    <h4 class="usa-graphic-list-heading">Step 2</h4>
+    <p>Mail it to your state.</p>
+    <img src="{{ .Site.BaseURL }}/assets/img/step-3.png">
+    <h4 class="usa-graphic-list-heading">Step 3</h4>
+    <p>Vote in the upcoming election!</p>
+  </div>
+  <!-- Steps END -->
+
+  <ul>
+    {{ range $.Site.Data.registration_forms }}
+    <li>
+      {{ . }}
```

Comment:
Ahh, this is a bit old, but in this context the `{{ . }}` will output everything that's in that map called `$.Site.Data.registration_forms`. Since this is calling the `$.Site.Data.registration_forms` property, it is looking at the `/data/registration_forms` directory and mapping the `*.toml` files in there into a map that looks similar to this.

```js
{
  "english": {
    "language": "English",
    "file_path": "/assets/files/*.pdf",
    "weight": 1
  },
  "spanish": { /* etc */ },
  /* etc etc */
}
```

In later commits, you'll see that I'm using things like `.file_paths` and `.language` directly or within a named context variable like `$forms.file_paths` in the `range` for-loop.

The documentation for this is a bit weird, but you can find it on the hugo site in the following places:

- https://gohugo.io/templates/go-templates/#iteration
- https://gohugo.io/templates/go-templates/#context-a-k-a-the-dot
- https://gohugo.io/extras/datafiles/
- https://golang.org/pkg/text/template/